test/jasmine/tests/image_test.js

```diff
@@ -535,4 +667,42 @@ describe('image hover:', function() {
             .then(done);
         });
     });
+
+    describe('for `image_labuda_droplets_source` defined by source', function() {
+        var mock = require('@mocks/image_labuda_droplets_source.json');
+        beforeAll(function() {
+            gd = createGraphDiv();
+        });
+
+        afterAll(destroyGraphDiv);
+
+        function _hover(x, y) {
+            var evt = { xpx: x, ypx: y };
+            return Fx.hover('graph', evt, 'xy');
+        }
+
+        [
+          ['x', '205'],
+          ['y', '125'],
+          ['color', '[78, 77, 83, 1]'],
+          ['color[0]', '78'],
```

Comment:
@emmanuelle the first elements in the arrays above are hovertemplate and the second elements are the expected values. As you can see, `color` is accessible!

Reply:
awesome! I made the change in `px.imshow` in my branch :-).